upstream-community-operators/ibm-application-gateway-operator/0.9.0/ibm-application-gateway-operator.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,394 @@
+
+
+    * developers
+
+    * security
+
+    * auditors
+
+    * devops
+
+    * etc
+
+
+    ##Sidecar
+
+    A Kubernetes application may consist of a number of different components.
+    These may include:
+
+
+    * Main application
+
+    * Security
+
+    * Logging
+
+    * Monitoring
+
+    * etc
+
+
+    Separating these components into their own container provides isolation and
+    encapsulation. 
+
+    Each container provides a single capability without needing to know the
+    intricate details 
+
+    of the other components.
+
+
+    In Kubernetes this pattern is known as a sidecar. Each sidecar container is
+    attached to the 
+
+    main application and provides a specific function.
+
+
+    The IBM Application Gateway supports the sidecar pattern such that it can be
+    configured to 
+
+    run alongside the main application providing authorization capabilities
+    without the application 
+
+    needing to know the details.
+
+
+    This operator when installed will also run as a Kubernetes admission
+    controller. If a deployment
+
+    object is created that contains the correct annotations the admission
+    controller will attempt to
+
+    create the sidecar IBM Application Gateway instance alongside the deployed
+    application.
+
+
+    For more information on the operator sidecar usage see 
+
+    [IBM Application Gateway
+    Sidecar](http://iamdevportal.us-east.mybluemix.net/iag/deployment/kubernetes/kubernetes-sidecar)
+
+
+    ##Usage
+
+    The official documentation for the [IBM Application Gateway
+    operator](http://iamdevportal.us-east.mybluemix.net/iag/deployment/kubernetes/kubernetes-operator)
+
+    describes the different usages of the operator in detail. There are also
+    Hello World style examples for both the 
+
+    [operator](http://iamdevportal.us-east.mybluemix.net/iag/hello-world/operator-kubernetes)
+    and
+    [sidecar](http://iamdevportal.us-east.mybluemix.net/iag/hello-world/sidecar-kubernetes)
+    usage.
+
+
+    To setup and test the IBM Application Gateway instance defined in the
+    example YAML below:
+
+
+    1. Install the operator and ensure that it is running by following the
+    installation instructions on this page.
+
+    2. Copy and apply the custom resource YAML
+
+       kubectl apply -f custom_resource.yaml
+       
+    3. Ensure that the IBM Application Gateway pod is running
+
+       kubectl get all
+       
+    4. Expose the IBM Application Gateway port
+
+       kubectl port-forward deployment.apps/iag-instance 30112:8443
+
+    5. In a browser access https://127.0.0.1:30112/static
+
+
+    6. The demo page is shown
+
+       This page includes the following pieces that have been added by the IBM Application Gateway:
+       * A JWT header has been added
+       * The IV-USER, IV-GROUPS and IV-SERVER-NAME headers have been added
+       * The JWT has been extracted
+
+
+
+
+  maturity: stable
+  version: 0.9.0
+  replaces: ''
```

Comment:
Since this is first version and you are not able to point to previous version, please remove this line. You can use it next time when you will replace current version. Thanks.